docs/ui-components/code.md

```diff
@@ -146,6 +146,16 @@ graph TD;
     C-->D;
 ```
 
+### Using a local mermaid library
+
+In order to use a local version of the mermaid library instead of one provided by jsDelivr, you can specify a `path` key in the mermaid configuration instead of a `version` key.
+
+```yaml
+mermaid:
+  # To load mermaid from a local file use the `path` key to specify the location of the library instead; e.g.
+  path: "/assets/js/mermaid.min.js"
+```
+
 *Note: for demonstration purposes, we've enabled mermaid on this site. It is still disabled by default, and users need to opt-in to use it.*
```

Comment:
Could you swap these two blocks? The note applies to the earlier header.

````suggestion
*Note: for demonstration purposes, we've enabled mermaid on this site. It is still disabled by default, and users need to opt-in to use it.*

### Using a local mermaid library

In order to use a local version of the mermaid library instead of one provided by jsDelivr, you can specify a `path` key in the mermaid configuration instead of a `version` key.

```yaml
mermaid:
  # To load mermaid from a local file use the `path` key to specify the location of the library instead; e.g.
  path: "/assets/js/mermaid.min.js"
```
````

Reply:
Sure!